rlbot_gui/gui.py

```diff
@@ -275,22 +316,13 @@ def is_chrome_installed():
     else:
         chm = eel.browsers.chm
         if hasattr(chm, 'get_instance_path'):
-            return chm.get_instance_path() is not None
-
+            return eel.browsers.chm.get_instance_path() is not None
         return chm.find_path() is not None
 
 
-def launch_eel(use_chrome):
+def launch_eel(browser_mode):
```

Comment:
This is what i noticed that you undid XD, but i stopped looking at this point and warned you :)

Reply:
yeah, PacificScienceScratcher seems to be broken on my merge test on own repo